### PR TITLE
(Feature) Dockerize

### PR DESCRIPTION
**Pull request by Stretch96**
* The Dockerfile is required to allow it to run within Dalmatian
* The docker-compose.yml is to help with local development

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+RUN addgroup --gid 999 --quiet app \
+  && adduser --gid 999 --uid 999 --quiet --disabled-password app
+
+# set rails environment
+ARG RAILS_ENV
+ENV RAILS_ENV=${RAILS_ENV:-production}
+ENV RACK_ENV=${RAILS_ENV:-production}
+
+# create and use application home
+ENV APP_NAME scheduling-dashboard
```
Comment (james) [inline Dockerfile]: Should probably rename this

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  app:
+    build:
+      context: .
+      args:
+        RAILS_ENV: development
+    user: root
+    env_file:
+      - docker-compose.common.env
+    volumes:
+      - .:/srv/scheduling-dashboard:cached
```
Comment (james) [inline docker-compose.yml]: and here

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,72 @@
+FROM ruby:2.6.5
```
Comment (james) [inline Dockerfile]: This is different to what is in .ruby-version, but we should use 2.6.5. I've tested it against 2.6.5, so we can either update .ruby-version in this PR, or I can make another one

File: Dockerfile [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+RUN bundle exec rake assets:precompile
+
+# make app own the project files
+RUN chown -R app:app $APP_PATH
+
+# run application as the app user
+USER app
+
+# expose public assets as a volume, this needs to happen after any changes to the filesystem
+VOLUME ["/srv/scheduling-dashboard/public"]
```
Comment (james) [inline Dockerfile]: another naming issue

File: Dockerfile (hunk 3 above)
Comment (Stretch96) [reply Dockerfile]: It says 2.6.5 in here: https://github.com/dxw/affordable-housing-monitoring/blob/develop/.ruby-version

Comment (james) [reply Dockerfile]: Doh, so it does. Ignore me!

Comment (Stretch96) [reply Dockerfile]: 😬

**Pull request merged**